recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
-{% set version = "1.4.3" %}
+{% set version = "1.5.2" %}
 
 package:
   name: typed-ast
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/t/typed_ast/typed_ast-{{ version }}.tar.gz
-  sha256: fb1bbeac803adea29cedd70781399c99138358c26d05fcbd23c13016b7f5ec65
+  sha256: 525a2d4088e70a9f75b08b3f87a51acc9cde640e19cc523c7e41aa355564ae27
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . -vv
-  skip: true  # [py2k]
+  skip: true  # [py36]
 
 requirements:
   build:
```

Comment:
You can remove `- python # [build_platform != target_platform]` form the `requirements/build`